### PR TITLE
tmux-cssh: init at 21750733c5b6fa2fe23b9e50ce69d8564f2f742a

### PR DESCRIPTION
**Pull request by zimbatm**
Currently based on master because the last release doesn't work with
makeWrapper

### CODE REVIEW

**Comment by sjourdois** [comment]
this commit contains also a lot of changes to packer

**Comment by zimbatm** [comment]
woops !

**Comment by zimbatm** [comment]
sorry, got overly exited, fixed

File: pkgs/tools/misc/tmux-cssh/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, makeWrapper, tmux }:
+
+stdenv.mkDerivation rec {
+  name = "tmux-cssh";
```
Comment (pSub) [inline pkgs/tools/misc/tmux-cssh/default.nix]: Please append a version or date to the name.

File: pkgs/tools/misc/tmux-cssh/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp tmux-cssh $out/bin/tmux-cssh
+    wrapProgram $out/bin/tmux-cssh --suffix PATH : ${tmux}/bin
+  '';
+
+  meta = {
+    homepage = https://github.com/dennishafemann/tmux-cssh;
+    description = "TMUX-C(luster)-SSH";
```
Comment (pSub) [inline pkgs/tools/misc/tmux-cssh/default.nix]: Please use a description that does not contain the program name.

File: pkgs/tools/misc/tmux-cssh/default.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, makeWrapper, tmux }:
+
+stdenv.mkDerivation rec {
```
Comment (pSub) [inline pkgs/tools/misc/tmux-cssh/default.nix]: Nitpick: Unless I missed something you don't need `rec` here.

**Comment by zimbatm** [comment]
@pSub : addressed all your comments and rebased in c46bc1f7e40f70b0dbd2d42f2b0c399881ec694e

**Pull request merged**